### ISSUE TITLE
Write user guide / tutorial

### ISSUE THREAD

**Issue by lam2mo** [Zamua/less-java#32]
Right now folks have to look at the examples to see what syntax constructs are valid in Less-Java. As more people get involved, we should have a short FAQ or tutorial or guide or quick reference or something that shows the basic syntax of Less-Java and helps on-board people who have had experience with Java or C.

**Comment by lam2mo** [comment]
@jennahorrall is working on this here: https://github.com/JMU-CS/less-java/wiki/Tutorial

**Comment by Zamua** [comment]
May also be worth moving the [Zamua updates](https://github.com/JMU-CS/less-java/wiki/Zamua-Updates) page somewhere else since it's a little out of place now. Or I suppose we can add pages for all of the contributors.

**Comment by lam2mo** [comment]
Yeah, I guess I had imagined making one for each student who does an honors thesis as a sort of record of what they've done. Maybe clean them up a bit afterwards and remove dates. Feel free to do that for yours if you like.

**Comment by Zamua** [comment]
Have we decided on how to write/host the documentation? 
I like [readthedocs](https://readthedocs.org/).

**Comment by lam2mo** [comment]
I'm not sure there's really a need for a separate host for now; the wiki formatting seems nice enough and it keeps the code and the documentation together in one place.

**Comment by Zamua** [comment]
Sounds good! Yeah on second thought the GitHub wiki formatting is pretty nice

**Comment by lam2mo** [comment]
@MrBartWolf make sure wrapper script options and `else if` are documented. Thanks!
